{
  "uid" : "557a83130c9b4a79",
  "name" : "Verify presence, visibility of sections, the dividing line and elements with text on the page",
  "fullName" : "tests.contacts_page_test.TestContactsPageStructure#test_cp_01_04_verify_page_structural_elements",
  "historyId" : "90770b19f2575efbe9bcd09069e52e01",
  "time" : {
    "start" : 1728784103730,
    "stop" : 1728784103730,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x560087dfc02a <unknown>\n#1 0x560087ae25e0 <unknown>\n#2 0x560087ada7eb <unknown>\n#3 0x560087acaa09 <unknown>\n#4 0x560087acc744 <unknown>\n#5 0x560087acacbd <unknown>\n#6 0x560087aca52f <unknown>\n#7 0x560087aca422 <unknown>\n#8 0x560087ac8493 <unknown>\n#9 0x560087ac8ada <unknown>\n#10 0x560087ae4f41 <unknown>\n#11 0x560087b768c5 <unknown>\n#12 0x560087b56b22 <unknown>\n#13 0x560087b75d7d <unknown>\n#14 0x560087b568c3 <unknown>\n#15 0x560087b246b3 <unknown>\n#16 0x560087b2568e <unknown>\n#17 0x560087dc6a2b <unknown>\n#18 0x560087dca9b1 <unknown>\n#19 0x560087db3225 <unknown>\n#20 0x560087dcb532 <unknown>\n#21 0x560087d9838f <unknown>\n#22 0x560087deaf28 <unknown>\n#23 0x560087deb0f3 <unknown>\n#24 0x560087dfae7c <unknown>\n#25 0x7f85b4094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"d86c956bb16218f743b8c40e02b105f8\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f7e99a0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x560087deb0f3 \\\\u003Cunknown>\\\\n#24 0x560087dfae7c \\\\u003Cunknown>\\\\n#25 0x7f85b4094ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x560087dfc02a <unknown>\nE       #1 0x560087ae25e0 <unknown>\nE       #2 0x560087ada7eb <unknown>\nE       #3 0x560087acaa09 <unknown>\nE       #4 0x560087acc744 <unknown>\nE       #5 0x560087acacbd <unknown>\nE       #6 0x560087aca52f <unknown>\nE       #7 0x560087aca422 <unknown>\nE       #8 0x560087ac8493 <unknown>\nE       #9 0x560087ac8ada <unknown>\nE       #10 0x560087ae4f41 <unknown>\nE       #11 0x560087b768c5 <unknown>\nE       #12 0x560087b56b22 <unknown>\nE       #13 0x560087b75d7d <unknown>\nE       #14 0x560087b568c3 <unknown>\nE       #15 0x560087b246b3 <unknown>\nE       #16 0x560087b2568e <unknown>\nE       #17 0x560087dc6a2b <unknown>\nE       #18 0x560087dca9b1 <unknown>\nE       #19 0x560087db3225 <unknown>\nE       #20 0x560087dcb532 <unknown>\nE       #21 0x560087d9838f <unknown>\nE       #22 0x560087deaf28 <unknown>\nE       #23 0x560087deb0f3 <unknown>\nE       #24 0x560087dfae7c <unknown>\nE       #25 0x7f85b4094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728784103731,
      "stop" : 1728784104516,
      "duration" : 785
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784104516,
      "stop" : 1728784104940,
      "duration" : 424
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x560087dfc02a <unknown>\n#1 0x560087ae25e0 <unknown>\n#2 0x560087ada7eb <unknown>\n#3 0x560087acaa09 <unknown>\n#4 0x560087acc744 <unknown>\n#5 0x560087acacbd <unknown>\n#6 0x560087aca52f <unknown>\n#7 0x560087aca422 <unknown>\n#8 0x560087ac8493 <unknown>\n#9 0x560087ac8ada <unknown>\n#10 0x560087ae4f41 <unknown>\n#11 0x560087b768c5 <unknown>\n#12 0x560087b56b22 <unknown>\n#13 0x560087b75d7d <unknown>\n#14 0x560087b568c3 <unknown>\n#15 0x560087b246b3 <unknown>\n#16 0x560087b2568e <unknown>\n#17 0x560087dc6a2b <unknown>\n#18 0x560087dca9b1 <unknown>\n#19 0x560087db3225 <unknown>\n#20 0x560087dcb532 <unknown>\n#21 0x560087d9838f <unknown>\n#22 0x560087deaf28 <unknown>\n#23 0x560087deb0f3 <unknown>\n#24 0x560087dfae7c <unknown>\n#25 0x7f85b4094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784104516,
        "stop" : 1728784104940,
        "duration" : 424
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x560087dfc02a <unknown>\n#1 0x560087ae25e0 <unknown>\n#2 0x560087ada7eb <unknown>\n#3 0x560087acaa09 <unknown>\n#4 0x560087acc744 <unknown>\n#5 0x560087acacbd <unknown>\n#6 0x560087aca52f <unknown>\n#7 0x560087aca422 <unknown>\n#8 0x560087ac8493 <unknown>\n#9 0x560087ac8ada <unknown>\n#10 0x560087ae4f41 <unknown>\n#11 0x560087b768c5 <unknown>\n#12 0x560087b56b22 <unknown>\n#13 0x560087b75d7d <unknown>\n#14 0x560087b568c3 <unknown>\n#15 0x560087b246b3 <unknown>\n#16 0x560087b2568e <unknown>\n#17 0x560087dc6a2b <unknown>\n#18 0x560087dca9b1 <unknown>\n#19 0x560087db3225 <unknown>\n#20 0x560087dcb532 <unknown>\n#21 0x560087d9838f <unknown>\n#22 0x560087deaf28 <unknown>\n#23 0x560087deb0f3 <unknown>\n#24 0x560087dfae7c <unknown>\n#25 0x7f85b4094ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"d86c956bb16218f743b8c40e02b105f8\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784104975,
      "stop" : 1728784105037,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contacts Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contacts_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContactsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contacts_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 28,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "ea6775d0e0e3762e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/ea6775d0e0e3762e",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f6e7c6a02a <unknown>\n#1 0x55f6e79505e0 <unknown>\n#2 0x55f6e79487eb <unknown>\n#3 0x55f6e7938a09 <unknown>\n#4 0x55f6e793a744 <unknown>\n#5 0x55f6e7938cbd <unknown>\n#6 0x55f6e793852f <unknown>\n#7 0x55f6e7938422 <unknown>\n#8 0x55f6e7936493 <unknown>\n#9 0x55f6e7936ada <unknown>\n#10 0x55f6e7952f41 <unknown>\n#11 0x55f6e79e48c5 <unknown>\n#12 0x55f6e79c4b22 <unknown>\n#13 0x55f6e79e3d7d <unknown>\n#14 0x55f6e79c48c3 <unknown>\n#15 0x55f6e79926b3 <unknown>\n#16 0x55f6e799368e <unknown>\n#17 0x55f6e7c34a2b <unknown>\n#18 0x55f6e7c389b1 <unknown>\n#19 0x55f6e7c21225 <unknown>\n#20 0x55f6e7c39532 <unknown>\n#21 0x55f6e7c0638f <unknown>\n#22 0x55f6e7c58f28 <unknown>\n#23 0x55f6e7c590f3 <unknown>\n#24 0x55f6e7c68e7c <unknown>\n#25 0x7f51d5894ac3 <unknown>",
        "time" : {
          "start" : 1728697217742,
          "stop" : 1728697217742,
          "duration" : 0
        }
      }, {
        "uid" : "1bbff3a37a045914",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/1bbff3a37a045914",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5624e11c802a <unknown>\n#1 0x5624e0eae5e0 <unknown>\n#2 0x5624e0ea67eb <unknown>\n#3 0x5624e0e96a09 <unknown>\n#4 0x5624e0e98744 <unknown>\n#5 0x5624e0e96cbd <unknown>\n#6 0x5624e0e9652f <unknown>\n#7 0x5624e0e96422 <unknown>\n#8 0x5624e0e94493 <unknown>\n#9 0x5624e0e94ada <unknown>\n#10 0x5624e0eb0f41 <unknown>\n#11 0x5624e0f428c5 <unknown>\n#12 0x5624e0f22b22 <unknown>\n#13 0x5624e0f41d7d <unknown>\n#14 0x5624e0f228c3 <unknown>\n#15 0x5624e0ef06b3 <unknown>\n#16 0x5624e0ef168e <unknown>\n#17 0x5624e1192a2b <unknown>\n#18 0x5624e11969b1 <unknown>\n#19 0x5624e117f225 <unknown>\n#20 0x5624e1197532 <unknown>\n#21 0x5624e116438f <unknown>\n#22 0x5624e11b6f28 <unknown>\n#23 0x5624e11b70f3 <unknown>\n#24 0x5624e11c6e7c <unknown>\n#25 0x7f5a47294ac3 <unknown>",
        "time" : {
          "start" : 1728610909089,
          "stop" : 1728610909089,
          "duration" : 0
        }
      }, {
        "uid" : "3e6203b967dff698",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/3e6203b967dff698",
        "status" : "passed",
        "time" : {
          "start" : 1728524555135,
          "stop" : 1728524555297,
          "duration" : 162
        }
      }, {
        "uid" : "fc3828d137c13128",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/fc3828d137c13128",
        "status" : "passed",
        "time" : {
          "start" : 1728438127746,
          "stop" : 1728438127908,
          "duration" : 162
        }
      }, {
        "uid" : "4593fd84f4bf7f9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/4593fd84f4bf7f9",
        "status" : "passed",
        "time" : {
          "start" : 1728351783761,
          "stop" : 1728351783922,
          "duration" : 161
        }
      }, {
        "uid" : "951ddb0bc07fdcb8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/951ddb0bc07fdcb8",
        "status" : "passed",
        "time" : {
          "start" : 1728265563893,
          "stop" : 1728265564054,
          "duration" : 161
        }
      }, {
        "uid" : "24d35023529f6a2b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/24d35023529f6a2b",
        "status" : "passed",
        "time" : {
          "start" : 1728179341864,
          "stop" : 1728179342024,
          "duration" : 160
        }
      }, {
        "uid" : "41a428c5a187994b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/41a428c5a187994b",
        "status" : "passed",
        "time" : {
          "start" : 1728092498458,
          "stop" : 1728092498620,
          "duration" : 162
        }
      }, {
        "uid" : "a5b593e0884f0f0c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/a5b593e0884f0f0c",
        "status" : "passed",
        "time" : {
          "start" : 1728006204308,
          "stop" : 1728006204472,
          "duration" : 164
        }
      }, {
        "uid" : "35aacbe09c120783",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/35aacbe09c120783",
        "status" : "passed",
        "time" : {
          "start" : 1727919791902,
          "stop" : 1727919792067,
          "duration" : 165
        }
      }, {
        "uid" : "eae5c9b35177d7bf",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/eae5c9b35177d7bf",
        "status" : "passed",
        "time" : {
          "start" : 1727833351550,
          "stop" : 1727833351715,
          "duration" : 165
        }
      }, {
        "uid" : "e84494e762b8fa14",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/e84494e762b8fa14",
        "status" : "passed",
        "time" : {
          "start" : 1727747487602,
          "stop" : 1727747487767,
          "duration" : 165
        }
      }, {
        "uid" : "f6a474f3fb1196d4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/f6a474f3fb1196d4",
        "status" : "passed",
        "time" : {
          "start" : 1727660769232,
          "stop" : 1727660769394,
          "duration" : 162
        }
      }, {
        "uid" : "d6530aaf1b504e1b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/d6530aaf1b504e1b",
        "status" : "passed",
        "time" : {
          "start" : 1727574568665,
          "stop" : 1727574568827,
          "duration" : 162
        }
      }, {
        "uid" : "fcd0cc1a71fea426",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/fcd0cc1a71fea426",
        "status" : "passed",
        "time" : {
          "start" : 1727487823053,
          "stop" : 1727487823214,
          "duration" : 161
        }
      }, {
        "uid" : "f0a2690fc05e880f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/f0a2690fc05e880f",
        "status" : "passed",
        "time" : {
          "start" : 1727401385970,
          "stop" : 1727401386130,
          "duration" : 160
        }
      }, {
        "uid" : "be219e26d0f33f44",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/be219e26d0f33f44",
        "status" : "passed",
        "time" : {
          "start" : 1727314932440,
          "stop" : 1727314932601,
          "duration" : 161
        }
      }, {
        "uid" : "f1e55ddfd1928ed5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/f1e55ddfd1928ed5",
        "status" : "passed",
        "time" : {
          "start" : 1727228631622,
          "stop" : 1727228631798,
          "duration" : 176
        }
      }, {
        "uid" : "8269375dd60b7b4e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/8269375dd60b7b4e",
        "status" : "passed",
        "time" : {
          "start" : 1727142193328,
          "stop" : 1727142193489,
          "duration" : 161
        }
      }, {
        "uid" : "d7adcae183579111",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/d7adcae183579111",
        "status" : "passed",
        "time" : {
          "start" : 1727055750469,
          "stop" : 1727055750623,
          "duration" : 154
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "557a83130c9b4a79.json",
  "parameterValues" : [ ]
}